lib/plugins/wallet/monerod/monerod.js

```diff
@@ -0,0 +1,206 @@
+const fs = require('fs')
+const path = require('path')
+const _ = require('lodash/fp')
+const jsonRpc = require('../../common/json-rpc')
+
+const { utils } = require('lamassu-coins')
+
+const blockchainUtils = require('../../../coin-utils')
+const BN = require('../../../bn')
+const E = require('../../../error')
+const { logger } = require('../../../blockchain/common')
+
+const cryptoRec = utils.getCryptoCurrency('XMR')
```

Comment:
Don't we have XMR as a constant on lamassu-coins? A bit nitpicky but it helps to remove strings from the code.